CorsixTH/Lua/world.lua

```diff
@@ -1205,13 +1205,8 @@ function World:onEndMonth()
     self:checkIfGameWon()
   end
 
-  -- Change population share for the hospitals, TODO according to reputation.
-  -- Since there are no competitors yet the player's hospital can be considered
-  -- to be fairly good no matter what it looks like, so after gbv.AllocDelay
-  -- months, change the share to half of the new people.
-  if self.month >= self.map.level_config.gbv.AllocDelay then
-    self:getLocalPlayerHospital().population = 0.5
-  end
+  local local_hospital = self:getLocalPlayerHospital();
```

Comment:
We want a calculation like this, yes, but I still believe we want it after the AllocDelay time. It is used to let the map maker decide when the hospitals no longer have 25 % each. For instance if on a certain map you start with a very low reputation, but still want the player to get some patients the first year to build reputation fast.